opentelemetry-quickstart/src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 quarkus.application.name=myservice
+quarkus.opentelemetry.enabled=true
```

Comment:
This gives warning in the log
```
16:44:32 WARN  traceId=, parentId=, spanId=, sampled= [io.qu.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.opentelemetry.tracer.exporter.otlp.headers" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo

16:44:32 WARN  traceId=, parentId=, spanId=, sampled= [io.qu.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.opentelemetry.tracer.exporter.otlp.endpoint" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
16:44:32 WARN  traceId=, parentId=, spanId=, sampled= [io.qu.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.opentelemetry.enabled" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
```

Reply:
Confirming re. the otel being enabled by default: https://github.com/quarkusio/quarkus/discussions/31635